string agent_name
geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
int width
int height
---
nav_msgs/Path path